metron-analytics/metron-profiler/src/test/config/zookeeper/readme-example-1/profiler.json

```diff
@@ -0,0 +1,17 @@
+{
+  "inputTopic": "indexing",
+  "profiles": [
+    {
+      "profile": "example1",
+      "foreach": "ip_src_addr",
+      "onlyif": "protocol == 'HTTP'",
+      "init": {
+        "total_bytes": 0.0
```

Comment:
Yes, the values within the 'init' block are executed as Stellar expressions.  See ProfileBuilderBolt.init(...).